fourkitchens/quiltme
# quiltme-128 send email to user

**Testing steps:**

After you choose an image, an email should be sent to the user.

The email sent flag will be set to 1.
